text/dotnet-core/rfcs/0004-extend-dotnet-sdk.md

```diff
@@ -69,14 +68,18 @@ would obviously use the cached resource.
 
 ## Implementation
 
-When constructing the .Net SDK dependency the pipeline should no longer strip
-any files out of the given dependency. It should instead take all of the files
-from the Microsoft release and compress them as we are doing for our current
-.Net SDK dependency. This will allow us to install a fully realized .Net SDK
-into a build only layer meaning that none of the extra SDK libraries or the
+The .Net SDK dependency referenced in the buildpack should be identical to the
+one provided by Microsoft. This will allow us to install a fully realized .Net
```

Comment:
For clarity with the dependency work happening: this means that the URI (and source URI) of the SDK dependency will be the Microsoft upstream URI, rather than the URI to a dependency that we host in the dep-server?

Reply:
I was trying to leave that implementation ambiguous for the time being cause I imagine it will start out as a reference to something the is hosted on dep-server and then move to the URI from Microsoft. I can state that it will still be on the dep-server.

Reply:
I think it's ok to leave this ambiguous. This RFC is about what the dependency _is_, not where it's hosted.